Lombiq.TrainingDemo/Controllers/CrossTenantServicesController.cs

```diff
@@ -24,6 +24,7 @@ namespace Lombiq.TrainingDemo.Controllers;
 // This is a controller just for the sake of easy demonstration, you can do the same thing anywhere. In the Index
 // action, we'll fetch content items from another tenant with the IContentManager service that you already know. This is
 // just an example though, really you can access any other service as well.
+[Route("[controller]/{action=Index}")]
```

Comment:
/CrossTenantServices?contentItemId=ID doesn't work anymore.

Reply:
Do we actually want this rule? The rule is generally saying that if there is an action level `[Route]` then there should be a Controller level `[Route]` aswell. So everything should be accessible via ControllerRoute/ActionRoute. We didn't really utilize this at all and I don't see the point why would we want for example an `api/somethings/paramteres` route to now be `ControllerRoute/ActionRoute/api/somethings/parameters`.
What do you think?

Reply:
Yeah, let's not use this, then.